server_admin/topics/user-federation/ldap.adoc

```diff
@@ -17,12 +17,12 @@ Selecting _ldap_ will bring you to the LDAP configuration page.
 
 By default, {project_name} will import users from LDAP into the local {project_name} user database. This copy of the user
 is either synchronized on demand, or through a periodic background task.
-The one exception to this is passwords.  Passwords are not imported and password validation is
+The one exception to this is passwords.  Passwords are not imported and password validation is always
 delegated to the LDAP server.  The benefits to this approach is that all {project_name} features will work as any extra
-per-user data that is needed can be stored locally.  This approach also reduces load on the LDAP server as uncached users are loaded
-from the {project_name} database the 2nd time they are accessed.  The only load your LDAP server will have is password validation.
-The downside to this approach is that when a user is first queried, this will require a {project_name} database insert.  The import will
-also have to be synchronized with your LDAP server as needed.
+per-user data that is needed can be stored locally.
+The downside to this approach is that when a user is first queried, this will require a {project_name} database insert. The import may
+also have to be synchronized with your LDAP server as needed. However import synchronization is not necessary in many cases as you can
+configure LDAP mappers in a way, that particular attributes of the user are preferably always read from the LDAP rather than from the DB.
```

Comment:
```s/However import synchronization is not necessary in many cases.../However, import synchronization is not necessary in the case the LDAP mappers are configured to always read particular attributes from the LDAP rather than from the DB./```

Reply:
Will change as you suggested.